shared/middleware/properties_api.rb

```diff
@@ -24,6 +24,8 @@ class PropertiesApi < Sinatra::Base
   get %r{/v3/(shared|user)-properties/([^/]+)$} do |endpoint, channel_id|
     dont_cache
     content_type :json
+    owner_id, _ = storage_decrypt_channel_id(channel_id)
+    not_authorized unless owner_id == storage_id('user')
```

Comment:
How about `owner_storage_id`?  At first glance I thought `owner_id` would be a user ID, but that doesn't seem to be the case.